packages/admin/cms-admin/src/pages/pageTree/useCopyPastePages/sendPages.tsx

```diff
@@ -397,6 +410,39 @@ function createPageTreeNodeIdReplacements(nodes: PageClipboard[]): ReplaceDepend
     return replacements;
 }
 
+function unhandledDependenciesFromDocument(
+    documentType: DocumentInterface,
+    document: GQLDocument,
+    { existingReplacements }: { existingReplacements: ReplaceDependencyObject[] },
+) {
+    const unhandledDependencies = documentType
+        .dependencies(document)
+        .filter(
+            (dependency) =>
+                !existingReplacements.some(
+                    (replacement) => replacement.originalId === dependency.id && replacement.type === dependency.targetGraphqlObjectType,
+                ),
+        );
+
+    return unhandledDependencies;
+}
+
+function createUndefinedReplacementsForDependencies(dependencies: BlockDependency[]) {
+    const existingReplacements = new Set();
+    const replacements: ReplaceDependencyObject[] = [];
+
+    for (const dependency of dependencies) {
+        const key = `${dependency.targetGraphqlObjectType}#${dependency.id}`;
+
+        if (!existingReplacements.has(key)) {
+            replacements.push({ type: "PageTreeNode", originalId: dependency.id, replaceWithId: undefined });
```

Comment:
you handle all dependencies, not just PageTreeNodes, but here you hardcode PageTreeNode. Is that really correct?

Reply:
No, that was a copy/paste mistake